Foreman 3.17.0 release notes

ogajduse opened the pull request:
I propose Release Notes for Foreman 3.17.0. Please let me know if I forgot to add something or if there is anything you'd change.

ogajduse committed:
Foreman 3.17.0 release notes

Lennonka commented on _includes/manuals/3.17/1.2_release_notes.md:
```diff
@@ -4,16 +4,230 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### PatternFly 5 UI Updates
```
```suggestion
#### PatternFly 5 UI updates
```

Lennonka commented on _includes/manuals/3.17/1.2_release_notes.md:
```diff
@@ -4,16 +4,230 @@ This section will be updated prior to the next release.
+#### PatternFly 5 UI Updates
+Continued migration to PatternFly 5 brings modernized notifications ([#38345](https://projects.theforeman.org/issues/38345)) and other UI components, improving the overall user experience with updated design patterns.
+
+#### Multi-Architecture iPXE Support
```
```suggestion
#### Multi-architecture iPXE support
```

Lennonka commented on _includes/manuals/3.17/1.2_release_notes.md:
```diff
@@ -4,16 +4,230 @@ This section will be updated prior to the next release.
+#### Multi-Architecture iPXE Support
+iPXE chain UEFI now supports multiple architectures ([#38300](https://projects.theforeman.org/issues/38300)), enabling proper provisioning of ARM64 and other non-x86_64 servers by delivering the correct iPXE binary for each architecture.
+
+#### Ubuntu 24 Support
```
```suggestion
#### Ubuntu 24 support
```

Lennonka commented on _includes/manuals/3.17/1.2_release_notes.md:
```diff
@@ -4,16 +4,230 @@ This section will be updated prior to the next release.
+iPXE chain UEFI now supports multiple architectures ([#38300](https://projects.theforeman.org/issues/38300)), enabling proper provisioning of ARM64 and other non-x86_64 servers by delivering the correct iPXE binary for each architecture.
+
+#### Ubuntu 24 Support
+Added support for Ubuntu 24 as a supported platform ([#38682](https://projects.theforeman.org/issues/38682)).
```
"Supported platform" for what exactly?

```suggestion
Added Ubuntu 24 as a supported platform for Foreman installer ([#38682](https://projects.theforeman.org/issues/38682)).
```

Lennonka replied:
I don't see the option to install Foreman on Ubuntu 24 documented in the [Quickstart](https://docs.theforeman.org/3.17/Quickstart/index-foreman-deb.html).

ogajduse replied:
Great catch, I have mistakenly added this to the Release Notes - Ubuntu 24 support is not a thing, yet.

ogajduse replied:
I've removed this item from Release Notes.

Lennonka commented on _includes/manuals/3.17/1.2_release_notes.md:
```diff
@@ -4,16 +4,230 @@ This section will be updated prior to the next release.
+
 ### Upgrade warnings
 
+#### Ruby 2.7 No Longer Supported
```
```suggestion
#### Ruby 2.7 no longer supported
```

Lennonka commented on _includes/manuals/3.17/1.2_release_notes.md:
```diff
@@ -4,16 +4,230 @@ This section will be updated prior to the next release.
 ### Deprecations
 
-### Release Notes
+#### Grub1 Removal
```
```suggestion
#### Grub1 removal
```

Lennonka reviewed (changes requested):
Please, use sentence case in headings.

Lennonka commented:
Oops, sorry, I missed that this PR is intended for the Foreman Manual. Feel free to dismiss my capitalization suggestions if not appropriate.

Lennonka commented on _includes/manuals/3.17/1.2_release_notes.md:
```diff
@@ -4,16 +4,228 @@ This section will be updated prior to the next release.
 ### Deprecations
 
-### Release Notes
+#### Grub1 removal
```
I think that this should go under _Upgrade warnings_ instead of _Deprecations_.

ogajduse replied:
Addressed.

Lennonka replied:
Thanks. Btw, I don't have permissions to resolve threads in this repo.

ogajduse replied:
I'd give you the permissions, but I don't have enough permissions to do that.

Lennonka reviewed (approved):
One structural suggestion. The rest LGTM. Will you get a tech ack?

ogajduse commented:
Thank you for your review, @Lennonka! I've addressed all your comments. Although this PR is not the Documentation one, it will serve as a solid base for the PR in the documentation project.
Could you rereview, please?

ogajduse commented:
> One structural suggestion. The rest LGTM. Will you get a tech ack?

I'll try to ping a few folks to get the tech ACK.

Lennonka commented on _includes/manuals/3.17/1.2_release_notes.md:
```diff
@@ -4,16 +4,226 @@ This section will be updated prior to the next release.
+Continued migration to PatternFly 5 brings modernized notifications ([#38345](https://projects.theforeman.org/issues/38345)) and other UI components, improving the overall user experience with updated design patterns.
+
+#### Multi-architecture iPXE support
+iPXE chain UEFI now supports multiple architectures ([#38300](https://projects.theforeman.org/issues/38300)), enabling proper provisioning of ARM64 and other non-x86_64 servers by delivering the correct iPXE binary for each architecture.
```
Terminology nitpick:
```suggestion
iPXE chain UEFI now supports multiple architectures ([#38300](https://projects.theforeman.org/issues/38300)), enabling proper provisioning of ARM64 and other non-x86_64 hosts by delivering the correct iPXE binary for each architecture.
```

Lennonka reviewed (approved):
LGTM style wise

ogajduse commented:
This PR was open for 5 days and received no feedback on the proposed Release Notes. If there is any feedback, please reach out to me in the community forum. We can still tweak the Release Notes before GA.
